src/tests/multihost/alltests/test_misc_proxy.py

```diff
@@ -345,10 +351,65 @@ def test_0002_bz1209483(self, multihost, backupsssdconf):
         assert "foo2:*:2001:2001::/home/foo2:/bin/bash" \
                in execute_cmd(multihost, "getent passwd -s "
                                          "sss foo2").stdout_text
-        ssh1 = SSHClient(client_e, username="foo2", password="Secret123")
+        ssh1 = SSHClient(client_e, username="foo2",
+                         password="Secret123")
         ssh1.close()
         execute_cmd(multihost, "userdel -rf foo2")
 
+    def test_bz1927195(self, multihost, backupsssdconf):
+        """
+        :title: sssd runs out of proxy child slots and
+          doesn't clear the counter for Active requests
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1927195
+        :id: 4c5b53e2-ec88-11ec-a41a-845cf3eff344
+        :steps:
+            1. Configure sssd with proxy
+            2. Try sssctl user-checks user@domain 10 times
+            3. Logs should not have error 'All available
+               child slots are full'
+        :expectedresults:
+            1. Should succeed
+            2. Should succeed
+            3. Should succeed
+        """
+        # When using authentication provider as proxy,
+        # User authentication suddenly stops working and
+        # starts working again only after restarting the sssd service.
+        tools = sssdTools(multihost.client[0])
+        domain_name = tools.get_domain_section_name()
+        domain_params = {'debug_level': '9',
+                         'id_provider': 'ldap',
+                         'proxy_pam_target': 'proxy_sleep',
+                         'auth_provider': 'proxy',
+                         'chpass_provider': 'ldap',
+                         'proxy_max_children': '10',
+                         'enumerate': 'false',
+                         'entry_cache_timeout': '300',
+                         'access_provider': 'proxy'}
+        tools.sssd_conf('domain/' + domain_name, domain_params)
+        tools.clear_sssd_cache()
+        client = multihost.client[0]
+        file_location = "/script/10_ssctl.sh"
+        client.transport.put_file(os.path.dirname(os.path.abspath(__file__))
+                                  + file_location, '/tmp/10_ssctl.sh')
+        execute_cmd(multihost, "chmod 755 /tmp/10_ssctl.sh")
+        execute_cmd(multihost, "sh /tmp/10_ssctl.sh")
+        time.sleep(3)
+        result = execute_cmd(multihost, "grep -c 'All available "
+                                        "child slots are full, "
+                                        "queuing request' "
+                                        "/var/log/sssd/sssd_example1.log")
+        assert result.stdout_text == '1\n'
+        time.sleep(60)
+        execute_cmd(multihost, "sed -i 's/11/1/g' /tmp/10_ssctl.sh")
```

Comment:
The command `sssctl user-checks foo1@example1` can be executed individually instead of updating the script.

Reply:
while running the 12th command, same time we need to check the sssd logs for the number of slot count. 

If we run this command directly :  sssctl user-checks foo1@example1 , we have to wait till  the command execution finishes. That's why i am updating the script